cmd/rp-rest/static/index.html

```diff
@@ -170,7 +170,7 @@ <h4 class="mt-10 text-lg font-medium tracking-wide uppercase leading-tight">Appl
         <div class="bg-white rounded-lg shadow-lg">
             <div class="px-4 py-8">
                 <button id="creditCardStatement" onclick="getCreditCardStatement()" class="h-24"><img src="img/icon-home-5.png"></button>
-                <h4 class="mt-10 text-lg font-medium tracking-wide uppercase leading-tight">Apply for ?????</h4>
+                <h4 class="mt-10 text-lg font-medium tracking-wide uppercase leading-tight">Apply for New Credit Card</h4>
```

Comment:
not sure it makes sense but we can discuss these details later

Reply:
yes, better than `?????` for now.